### DISCUSSION TITLE
How select achieves the same effect as join

### DISCUSSION THREAD

**Discussion by hanyisong** [databendlabs/databend#5614] [Q&A] [answered]
```
mysql> select * from t0, t1;
ERROR 1105 (HY000): Code: 1002, displayText = Unimplemented SELECT JOIN yet..
```

**Comment by Xuanwo** [comment]
No idea so far. I am inviting @leiysky & @andylokandy to take a look.

**Comment by bohutang** [comment]
It needs enable the new planner for your session:
set enable_planner_v2 = 1;

https://github.com/datafuselabs/databend/blob/main/tests/suites/0_stateless/20%2B_others/20_0001_planner_v2.sql

**Comment by leiysky** [comment, marked as answer]
> ```
> mysql> select * from t0, t1;
> ERROR 1105 (HY000): Code: 1002, displayText = Unimplemented SELECT JOIN yet..
> ```

As @BohuTANG said, you need to enable the new planner. And we only support inner join with explicit join condition for now, it allows you to write a join query as:

```sql
select * from t inner join t1 on t.a = t1.a;
select * from t natural join t1;
select * from t inner join t1 using(a);
```

We are implementing the rest join types, you can track it in #5498.

**Comment by xudong963** [comment]
I'll support  the `select * from t0, t1;`  syntax next week in the new planner, after ready, I'll ping you @hanyisong

**Reply by Xuanwo** [reply]
Let's create an issue (or comment on the existing one) with the following information:

- Link to this discussion.
- Ping @hanyisong to notify.

**Reply by leiysky** [reply]
Here's the related issue, we don't need to create another one. https://github.com/datafuselabs/databend/issues/5499
